test/performance/scripts/create-outgoing-payments.js

```diff
@@ -16,25 +19,42 @@ const CLOUD_NINE_WALLET_ADDRESS =
   'https://cloud-nine-wallet-backend/accounts/gfranklin'
 const HAPPY_LIFE_BANK_WALLET_ADDRESS =
   'https://happy-life-bank-backend/accounts/pfry'
+const SIGNATURE_SECRET = 'iyIgCprjb9uL8wFckR+pLEkJWMB7FJhgkvqhTQR/964='
+const SIGNATURE_VERSION = '1'
+
+function generateSignedHeaders(requestPayload) {
+  const timestamp = Date.now()
+  const payload = `${timestamp}.${canonicalize(requestPayload)}`
+  const hmac = createHMAC('sha256', SIGNATURE_SECRET)
+  hmac.update(payload)
+  const digest = hmac.digest('hex')
+
+  return {
+    'Content-Type': 'application/json',
+    signature: `t=${timestamp}, v${SIGNATURE_VERSION}=${digest}`
+  }
+}
 
 export function setup() {
-  const c9WalletAddressesRes = http.post(
-    CLOUD_NINE_GQL_ENDPOINT,
-    JSON.stringify({
-      query: `
-    query GetWalletAddresses {
-      walletAddresses {
-        edges {
-          node {
-            id
-            url
+  const query = {
+    query: `
+      query GetWalletAddresses {
+        walletAddresses {
+          edges {
+            node {
+              id
+              url
+            }
           }
         }
       }
-    }
-  `
-    }),
-    { headers: HEADERS }
+    `
+  }
+
+  const c9WalletAddressesRes = http.post(
+    CLOUD_NINE_GQL_ENDPOINT,
+    JSON.stringify(query),
+    { headers: generateSignedHeaders(query) }
```

Comment:
nit: we could turn making the api request into a function so we don't need to duplicate `JSON.stringify(query)` + `{ headers: generateSignedHeaders(query) }`

Reply:
Made a `request` that forms the headers and query, calls http.post, and returns the data or errors if unsuccessful.